mlflow/R/mlflow/R/model-serve.R

```diff
@@ -157,6 +156,47 @@ serve_handlers <- function(host, port, ...) {
         ))
       )
     },
+    "^/ping" = function(req, model) {
```

Comment:
Could we document the fact that the previous endpoints are deprecated (to be removed in MLflow 2.0)?

Reply:
yes we should do that, I'll update the PR.